x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/search_session_tour.tsx

```diff
@@ -36,13 +40,16 @@ export function useSearchSessionTour(
     if (state === SearchSessionState.Loading) {
       if (!safeHas(storage, TOUR_TAKING_TOO_LONG_STEP_KEY)) {
         timeoutHandle = window.setTimeout(() => {
+          usageCollector?.trackSessionIndicatorTourLoading();
           searchSessionIndicatorRef.openPopover();
         }, TOUR_TAKING_TOO_LONG_TIMEOUT);
       }
     }
 
     if (state === SearchSessionState.Restored) {
+      usageCollector?.trackSessionIsRestored();
       if (!safeHas(storage, TOUR_RESTORE_STEP_KEY)) {
+        usageCollector?.trackSessionIndicatorTourRestored();
         searchSessionIndicatorRef.openPopover();
```

Comment:
I think it could be beneficial to report when the indicator disables itself due to a completion timeout as well